specification/compute/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/examples/ContainerGroupsListAll.json

```diff
@@ -8,50 +8,71 @@
       "body": {
         "value": [
           {
-            "id": "/subscriptions/subid/resourceGroups/demo/providers/Microsoft.ContainerInstance/containerGroups/demo1",
-            "location": "west us",
-            "name": "demo1",
-            "properties": {
```

Comment:
why did you remove this nesting? This was correct (or at least according to the Swagger) before! Are you sure that your API returns *this*???